### PR TITLE
LRQA-66638

### PR DESCRIPTION
**Pull request by guivilarim**

### CODE REVIEW

**Comment by guivilarim** [comment]
ci:test

**Comment by guivilarim** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/DataEngine.macro [hunk 1]
```diff
@@ -1378,6 +1378,50 @@ definition {
 		Click(locator1 = "FormViewBuilder#CREATE_NEW_FIELD_SETS");
 	}
 
+	macro gotoEditFieldset{
+		DataEngine.delete(
+			fieldsetLabel = "${fieldsetName}"
+		);
```
Comment (emanuelfisilva) [inline portal-web/test/functional/com/liferay/portalweb/macros/DataEngine.macro]: You could let this end of parenthesis and semi-colon on the upper line

File: portal-web/test/functional/com/liferay/portalweb/macros/DataEngine.macro [hunk 2]
```diff
@@ -1378,6 +1378,50 @@ definition {
...
+		DataEngine.gotoFieldsetItemEllipsis(fieldsetName = "${fieldsetName}");
+		MenuItem.click(menuItem = "Edit");
```
Comment (emanuelfisilva) [inline portal-web/test/functional/com/liferay/portalweb/macros/DataEngine.macro:1384]: Please, remember to insert a line break between actions to increase readability.

**Review by emanuelfisilva — CHANGES_REQUESTED** [review]
Your code is not compiling:
```
Unexpected closing boundary at:
     [exec]     /home/emanuel/lportal/portal-master/liferay-portal/modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:492
     [exec]         webContentTitle = "Web Content Title");
```
Your last commit _e3351886ce7833b3015c5cca8a43816cc22cc18e_ deletes the test **DefaultTitleIsDisplayedUponLocaleChange**.

Also, please run SF on your branch.

Thanks.

**Comment by guivilarim** [comment]
ci:test:sf

**Comment by emanuelfisilva** [comment]
Closing this as the development of this task is blocked.
Please, rebase and resend when we can continue with this.

**Pull request closed without merging**